.github/workflows/sanity-check.yml

```diff
@@ -1,16 +1,52 @@
 name: Commit message format check
-
-on: [pull_request]
+on:
+  # ATTENTION: See https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
+  # re security implications of using this trigger; in particular, no code from PR branches must
+  # be executed in any flows triggered by it
+  pull_request_target:
+    branches:
+      - main
+      - 1.*
+      - 2.*
 
 jobs:
   build:
     name: Commit message
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v3
       - name: Commit messages in format DBZ-xxx
+        id: check
         env:
-          BASE_REF: ${{ github.base_ref }}
+          pull_request_number: ${{ github.event.pull_request.number }}
         run: |
-          ! git log --format='format:%s' refs/remotes/origin/$BASE_REF.. | tail -n +2 | grep -Ev '^(DBZ-[[:digit:]]+)|(\[release\])|(\[jenkins-jobs\]) '
+          COMMIT_MSGS=$(curl --silent -X "GET" https://api.github.com/repos/debezium/debezium-connector-db2/pulls/$pull_request_number/commits | jq '.[] | {message: .commit.message}' | jq '.message' | cut -c 2-)
```

Comment:
@jpechane My bad I used the wrong Db2 repo instead of vitess! Sorry for the confusion. Could you directly commit it?